### DISCUSSION TITLE
Ionic and Angular Paypal integration

### DISCUSSION THREAD

**Discussion by marshall86** [ionic-team/capacitor#4616] [Q&A]
Hi everyone,

as per title, I need to integrate Paypal in my ionic + capacitor app. 
Considering that the ionic native Paypal plugin is archived and the recommended Braintree plugin is not in development anymore since a couple of years now, what's the best approach?

Thanks

**Comment by thomasvidas** [comment]
You can try either the [Paypal JS SDK](https://developer.paypal.com/docs/business/javascript-sdk/javascript-sdk-reference/) if that fits your use case, or you can use the Braintree [Android](https://github.com/braintree/braintree_android) and [iOS](https://github.com/braintree/braintree_ios) SDKs and follow our [plugin guide](https://capacitorjs.com/docs/plugins/creating-plugins).

**Reply by marshall86** [reply]
Hi, 

thanks for your feedback. I think I would go with the first option as I'm not very familiar with native code, i don't even know from where to start :D

Regarding the Paypal JS SDk, what it is the best option to integrate it in my ionic app? 

Thanks for your support
